lib/types.js

```diff
@@ -55,6 +55,19 @@ var LOGICAL_TYPE = null;
 // to be able to reference names (i.e. for branches) during instantiation.
 var UNDERLYING_TYPES = [];
 
+
+// range of values represented by 32 bit signed int
+const MAX_INT = 2 ** 31 - 1;
+const MIN_INT = 2 ** 31 * -1;
+
+// range of values represented by 64 bit signed int
+const MAX_LONG = 2 ** 63 - 1;
+const MIN_LONG = 2 ** 63 * -1;
```

Comment:
This isn't safe unfortunately, see https://github.com/mtth/avsc/wiki/Advanced-usage#custom-long-types. Here you can use `isSafeLong` instead.